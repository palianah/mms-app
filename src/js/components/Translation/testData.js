// @flow

window.app = {
  translations: {
    EN: {
      "App": {
        "Name": "Github Issues"
      }
    },
    DE: {
      "App": {
        "Name": "DE Github Issues"
      }
    },
  },
  languages: ['EN', 'DE'],
  curLang: 'EN',
  defLang: 'EN'
};
