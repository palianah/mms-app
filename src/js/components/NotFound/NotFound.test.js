import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { MemoryRouter } from 'react-router'
import NotFound from './NotFound';
import '../Translation/testData';

configure({ adapter: new Adapter() });

describe('<NotFound />', () => {
  it('Renders without crashing', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toHaveLength(1);
  });

  it('Matches snapshot', () => {
    expect(renderer.create(<MemoryRouter><NotFound /></MemoryRouter>).toJSON()).toMatchSnapshot();
  });
});
