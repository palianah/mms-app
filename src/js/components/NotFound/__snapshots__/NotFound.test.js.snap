// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> Matches snapshot 1`] = `
<section
  className="NotFound"
>
  <p
    className="NotFound__msg"
  >
    Page or resource not found
  </p>
  <p
    className="NotFound__submsg"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      Back to Github Issues
    </a>
  </p>
</section>
`;
